client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/support-nudge/style.scss

```diff
@@ -0,0 +1,8 @@
+.site-migration-instructions-support-nudge {
+	margin-inline-start: auto;
+	font-size: 0.75rem;
+
+	.components-button {
```

Comment:
``` suggestion
	.some-more-specific-class {
```

Reply:
Thanks for flagging! I got caught up on the fact that the default button font size is 1px larger than the other text in the nudge.

Done in f9a4479ed9c0e2fd46095f717cadeedb9da2bfe9.